packages/manager/modules/bm-server-components/src/general-information/installation/gabarit/server-installation-gabarit.controller.js

```diff
@@ -0,0 +1,511 @@
+      useSpla: false,
+      validForm: true,
+    };
+
+    this.Server.getPartitionSchemesByPriority(
+      this.$stateParams.productId,
+      this.$scope.installation.selectGabarit.id,
+    )
+      .then((response) => {
+        this.$scope.installation.partitionSchemesList = response;
+        [this.$scope.installation.selectPartitionScheme] = response;
+        return response;
+      })
+      .then((response) =>
+        this.Server.getPartitionSchemeHardwareRaid(
+          this.$stateParams.productId,
+          this.$scope.installation.selectGabarit.id,
+          response[0],
+        ),
+      )
+      .then((response) => {
+        if (response[0]) {
+          [tempHardwareRaid] = response;
+          return this.Server.getHardwareRaidProfile(
+            this.$stateParams.productId,
+          );
+        }
+        return null;
+      })
+      .catch((error) => {
+        if (this.Server.isHardRaidLocationError(error)) {
+          this.$scope.errorGab.ws = this.$translate.instant(
+            'server_configuration_installation_ovh_step1_hardwareRaid_wrong_location',
+          );
+          this.$scope.informations.hardwareRaidCompatible = false;
+        } else if (this.Server.isHardRaidUnavailableError(error)) {
+          this.$scope.errorGab.ws = this.$translate.instant(
+            'server_configuration_installation_gabarit_step2_hardwareRaid_incompatible_noHardwareRaid',
+          );
+          this.$scope.informations.hardwareRaidCompatible = false;
+        } else {
+          this.$q.reject(error);
+        }
+      })
+      .then(() => {
+        if (tempHardwareRaid) {
+          this.$scope.informations.hardwareRaid = tempHardwareRaid;
+          if (!this.$scope.installation.server.raidController) {
+            this.$scope.errorGab.ws = this.$translate.instant(
+              'server_configuration_installation_gabarit_step2_hardwareRaid_incompatible_noHardwareRaid',
+            );
+            this.$scope.informations.hardwareRaidCompatible = false;
+          } else {
+            this.$scope.informations.hardwareRaid.disks = ServerInstallationGabaritCtrl.getDisks(
+              this.$scope.informations.hardwareRaid.disks,
+            );
+            if (
+              this.$scope.installation.server.nbPhysicalDisk <
+              this.$scope.informations.hardwareRaid.disks.length
+            ) {
+              this.$scope.errorGab.ws = this.$translate.instant(
+                'server_configuration_installation_gabarit_step2_hardwareRaid_incompatible_disks',
+                {
+                  t0: this.$scope.installation.server.nbPhysicalDisk,
+                  t1: this.$scope.informations.hardwareRaid.disks.length,
+                },
+              );
+              this.$scope.informations.hardwareRaidCompatible = false;
+            }
+          }
+        }
+      })
+      .catch((error) => {
+        this.$scope.resetAction();
```

Comment:
Can this be replaced by this.goBack(); ?

Reply:
it's fixed